### ISSUE TITLE
Is there a way to disable inline styles?

### ISSUE THREAD

**Issue by vkoves** [c-w/mathquill4quill#35]
I've recently setup mathquill4quill in an Angular project, and the button styling looks off due to how our app's default styles interface with mathquill4quill's styles. Here's a screenshot:

![Screenshot from 2020-02-13 13-57-22](https://user-images.githubusercontent.com/3187531/74473239-dd06d800-4e68-11ea-8ee2-7db4c086b50f.png)

I was looking to override the mathquill4quill styles, but I noticed that they are all inline, making them really hard to override or modify. Is there a way to disable these inline styles so that I can custom style the operator buttons without using `!important`?

**Comment by c-w** [comment]
@vkoves I opened https://github.com/c-w/mathquill4quill/pull/36 to make the styles injectable via a constructor argument:

```js
const enableMathQuillFormulaAuthoring = window.mathquill4quill();

const quill = new Quill("#editor");

enableMathQuillFormulaAuthoring(quill, {
  styles: {
    mathquillInput: "...some css string here...",
    operatorButton: "...some css string here...",
    operatorContainer: "...some css string here...",
    latextInput: "...some css string here..."
  }
});
```

Could you take a look at the pull request and let me know if this fulfills your requirements?

**Comment by c-w** [comment]
A fix for this has been released in [v2.0.0](https://github.com/c-w/mathquill4quill/releases/tag/v2.0.0).
